search-service/src/main/kotlin/com/egm/stellio/search/util/AttributeInstanceUtils.kt

```diff
@@ -15,3 +20,20 @@ fun valueToStringOrNull(value: Any): String? =
         is String -> value
         else -> null
     }
+
+fun extractAttributeInstanceFromParsedPayload(
```

Comment:
this function is actually doing three things (and two are hidden):
- extract the attribute from the payload (that could go into the shared `JsonLdUtils`)
- insert an instance id (highly specific)
- serialize the whole (could be done separately by the caller)